.tx/config

```diff
@@ -0,0 +1,27 @@
+[main]
+host = https://www.transifex.com
+
+[beavy.server]
+file_filter = translations/<lang>/beavy_server.json
+source_file = var/server-messages/beavy-server.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.web]
+file_filter = translations/<lang>/beavy_web.json
+source_file = var/react-intl-messages/beavy-core.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.hacker-news-app]
```

Comment:
I agree with that. But can we do that in transifex? I looked briefly and it didn't seem possible within their naming conventions for the config file. This is from [their documentation](http://docs.transifex.com/client/config/), where they explain that line in the config file:

> [project_slug.resource_slug] : your project and resource 'slugs', separated by a dot. These are the strings found on your project's URL and together they uniquely identify your translatable resource.

So they only seem to allow for a project name, and in their setup each "project" has its own config file, and within each project different resources.